### DISCUSSION TITLE
Decimal Serializer Field should be able to automatically round

### DISCUSSION THREAD

**Discussion by mfit** [encode/django-rest-framework#8072] [Ideas & Suggestions]
Decimal Serializer Field shoudl be able to automatically round to desired precision.

Instead of raising validation.

This behavior could be optional / opt-in.

**Comment by blueshack112** [comment]
Same problem here, I have an application that heavily deals with pricing data. It honestly feels like I'm writing java again when I look at how many .quantize (in the backend) and lodash.round or .toFixed functions I am using just to make sure no number ever goes beyond 2 precision points. 

This seems so obvious and is much needed
